javalib/src/main/scala/java/util/logging/Handler.scala

```diff
@@ -0,0 +1,54 @@
+package java.util.logging
+
+import java.nio.charset.{Charset, UnsupportedCharsetException}
+
+abstract class Handler protected () {
+  private[this] var level:Option[Level] = Some(Level.ALL)
+  private[this] var filter:Option[Filter] = None
+  private[this] var formatter:Option[Formatter] = None
+  private[this] var encoding:Option[String] = None
+  private[this] var errorManager:Option[ErrorManager] = None
```

Comment:
Missing space between the `:` and `Option[ErrorManager]`. The same applies for the method types bellow.